src/style.css

```diff
@@ -1,3 +1,76 @@
+/* Reset https://www.joshwcomeau.com/css/custom-css-reset/ */
+
+/*
+  1. Use a more-intuitive box-sizing model.
+*/
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
```

Comment:
we should add this to the starter as well, not just in the final solution. it's actually a great point to discuss with beginners, I remember when I started with html/css this was one of the things that always confused me

Reply:
@andapop  feel free to make some of these changes also in this branch `starterImprovement` that has an open PR. We can pieces of this CSS also for the starter, just like @alexnm  said, and also we can update the HTML with your new changes. :)

Reply:
UPDATE: `starterImprovement` branch is now merged so maybe we can create a new branch for the above comment